class.jetpack-twitter-cards.php

```diff
@@ -196,9 +198,12 @@ static function sharing_global_options() {
 	static function site_tag() {
 		$site_tag = get_option( 'jetpack-twitter-cards-site-tag' );
 		if ( empty( $site_tag ) ) {
-			$site_tag = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'wordpressdotcom' : 'jetpack';
+			return;
+		} else if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
Also, this conditional will never work properly, as the middle branch will always override on wpcom sites, never letting the default option get through, and the first block will always return empty if it's empty, regardless of whether it's jetpack or wpcom, so the wpcom default wont happen ...